File: need4spd/euler_13.py
numlist=list()
sampleFile=open("euler_13_problem.txt", "r")
for l in sampleFile.readlines():
	l=l.rstrip('\n')
	numlist.append(int(l))

sumofnumbers=sum(numlist)

print(str(sumofnumbers)[0:10])

File: need4spd/euler_13.rb
numlist=[]
File.open("euler_13_problem.txt") do |file|
	while line=file.gets
		numlist.push(line)
	end
end

sum=0
numlist.each do |num|
	sum+=num.to_i
end

puts sum.to_s.slice(0, 10)